src/sync/branches.ts

```diff
@@ -13,6 +13,7 @@ export default async (logger: LoggerWithTarget, github: GitHubAPI, newGithub: Gi
 	logger.info("Syncing branches: started");
 
 	const useNewGHClient = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_FOR_BRANCHES, false, jiraHost);
+	const useNewGHPrUrl = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_PULL_REQUEST_URL_FORMAT, false, jiraHost);
```

Comment:
This was placed here instead of inside the transform/branches.ts files so I had access to Jira host, but also there was a bunch of extra conversions down the chain to async just to use the flag..... 
TLDR; was trying to keep the tidy up simpler.